lib/loggers.js

```diff
@@ -80,8 +80,8 @@ BufferedMetricsLogger.prototype.increment = function(key, value, tags, timestamp
     }
 };
 
-BufferedMetricsLogger.prototype.histogram = function(key, value, tags, timestampInMillis) {
-    this.addPoint(Histogram, key, value, tags, timestampInMillis);
+BufferedMetricsLogger.prototype.histogram = function(key, value, tags, timestampInMillis, options) {
```

Comment:
Could we add an optional `{}` value for `options`?

Reply:
Of course. Added 👍